CDCgov/prime-reportstream
# Add ordering_provider_name_type_code and testing_lab_name_type_code to sender schemas

This PR adds two fields to the sender schemas: ordering_provider_name_type_code and testing_lab_name_type_code. The purpose of these fields is to add values to OBR-16.10 and OBX-23.2 respectively. These fields were requested by PA as part of their onboarding but will be sent with all STLT reporting with this PR. Both fields are type codes and should only be present when their primary fields(OBR-16.1 and OBX-23.1) have values.

Test Steps:
1. Pull an HL7 and CSV file for testing
     - sample HL7 file: `./prime sender-files --env prod --message-id 20230124100151`
     - sample CSV file: `./prime sender-files --env prod --message-id C80402368X+33+20230125`
2. add settings for PA: [pa-phd.txt](https://github.com/CDCgov/prime-reportstream/files/10513907/pa-phd.txt)
3. run curl to submit test files to docker
4. check output files for OBR-16.10 and OBX-23.2. Both values should be "L"

## Checklist

### Testing
